src/scripts/clientUtils.js

```diff
@@ -19,12 +19,6 @@ export async function serverAddress() {
         return Promise.resolve(apiClient.serverAddress());
     }
 
-    const current = await ServerConnections.getAvailableServers().then(servers => {
-        if (servers.length !== 0) {
-            return Promise.resolve(servers[0].ManualAddress);
-        }
-    });
-
     // TODO this makes things faster but it also blocks the wizard in some scenarios
     // if (current) return Promise.resolve(current);
```

Comment:
seems unneeded

Reply:
You're probably right, but I'd like to keep the scope of this PR as limited as possible. :smile: